### PR TITLE
[WIN CI] Removing unused directories

### PR DESCRIPTION
**Pull request by rasapala**
### 🛠 Summary

JIRA CVS-157887
Looking for active jenkins workspace and comparing the list with bazel c:\PR-xxxx on a node.
Removing any inactive PR-xxxx directory to cleanup jenkins nodes.

### 🧪 Checklist

- [ ] Unit tests added.
- [ ] The documentation updated.
- [ ] Change follows security best practices.
``

### CODE REVIEW

File: ci/loadWin.groovy [hunk 1]
```diff
@@ -1,3 +1,60 @@
...
+        def found = false
+        for (int j = 0; j < existing_wr.size(); j++) {
+            if (existing_pr[i].toLowerCase() == existing_wr[j].toLowerCase()) {
+                found = true
+                break
+            }
+            // Part of output contains the command that was run
+            if (existing_pr[i].toLowerCase().contains("grep")) {
+                found = true
+                break
```
Comment (mzegla) [inline ci/loadWin.groovy:37]: Looks hacky 😕

File: ci/loadWin.groovy [hunk 2]
```diff
@@ -1,3 +1,60 @@
...
+    status = bat(returnStatus: true, script: command)
+    if ( status != 0) {
+        println "No PR-XXXX detected for cleanup."
+        return
+    }
+
+    def existing_pr_string = bat(returnStatus: false, returnStdout: true, script: command)
+
+    println existing_pr_string
+    def existing_pr = existing_pr_string.split(/\n/)
```
Comment (mzegla) [inline ci/loadWin.groovy:24]:
```suggestion
    def existing_prs = existing_pr_string.split(/\n/)
```
?

File: ci/loadWin.groovy (hunk 1 above)
Comment (rasapala) [reply ci/loadWin.groovy:37]: Unfortunately no other way to do it works.

**Review by mzegla — APPROVED** [review]

File: docs/windows_developer_guide.md [hunk 3]
```diff
@@ -109,6 +116,14 @@ set BAZEL_VC_FULL_VERSION=14.29.30133
 bazel --output_user_root=C:/b_tmp build --config=windows --jobs=%NUMBER_OF_PROCESSORS% --subcommands --verbose_failures //src:ovms 2>&1 | tee compilation.log
 ```
 
+# Set python paths
+```
+set "copyPyovms=cp %cd%\bazel-out\x64_windows-opt\bin\src\python\binding\pyovms.so %cd%\bazel-out\x64_windows-opt\bin\src\python\binding\pyovms.pyd"
```
[the remark this answers is not in the document]
Comment (rasapala) [reply docs/windows_developer_guide.md:121]: This is required for pythonNodes.
Will be changed in future.

**Review by dtrawins — APPROVED** [review]

**Pull request merged**